src/components/postElements/body/view/postBodyView.js

```diff
@@ -47,9 +47,38 @@ class PostBody extends PureComponent {
         handleOnPostPress(href);
       }
     } else {
-      Linking.canOpenURL(href).then((supported) => {
+      this._handleBrowserLink(href);
+    }
+  };
+
+  _handleBrowserLink = async (url) => {
+    if (!url) return;
+
+    let author;
+    let permlink;
+    const { intl } = this.props;
+
+    if (
```

Comment:
we should handle these url at `markdownToHtml.js` we can change as our app url but for now I'll approve maybe next time we can move these to that file. just quick nitpick otherwise approved 👍